app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -71,16 +96,39 @@ def emails
       emails
     end
 
-    def account_reuse_report
-      @account_reuse_report ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
+    def total_user_count_report
+      @total_user_count_report ||= Reporting::TotalUserCountReport.new(report_date)
     end
 
     def account_deletion_rate_report
       @account_deletion_rate_report ||= Reporting::AccountDeletionRateReport.new(report_date)
     end
 
-    def total_user_count_report
-      @total_user_count_report ||= Reporting::TotalUserCountReport.new(report_date)
+    def account_reuse_report
+      @account_reuse_report ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
+    end
+
+    def monthly_proofing_report
+      prepared_report = []
+
+      CSV.parse(get_raw_proofing_report).each do |row|
+        unless row.first.start_with?('report')
+          prepared_report << row
+        end
+      end
```

Comment:
I think it's silly of us to re-parse a table like this when we have control over the class that builds it. Let's add an `as_csv` method to the report and have its `to_csv` call into that?